PyPDF2/_page.py

```diff
@@ -291,6 +291,17 @@ def hash_value_data(self) -> bytes:
         data += b"%d" % id(self)
         return data
 
+    @property
+    def user_unit(self) -> float:
+        """
+        A read-only positive number giving the size of default user space units.
+
+        It is in multiples of 1 ⁄ 72 inch. Hence a value of 1 means a user space
+        unit is 1/72 inch, and a value of 3 means that a user space unit is
+        3 / 72 inch.
```

Comment:
Looks weird to me to have three different notations for fractional numbers in the same block

```suggestion
        It is in multiples of 1/72 inch. Hence a value of 1 means a user space
        unit is 1/72 inch, and a value of 3 means that a user space unit is
        3/72 inch.
```